pkg/util/labels/labels.go

```diff
@@ -0,0 +1,12 @@
+package labels
```

Comment:
Maybe:

```suggestion
package support
```

Reply:
I would prefer we call this package as runtime, similar to trainjob utils: https://github.com/kubeflow/trainer/blob/master/pkg/util/trainjob/trainjob.go

Reply:
That's a good idea.